docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,727 @@
+.. _integrations-pytorch-lightning:
+
 Neptune-PyTorch Lightning Integration
 =====================================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
+What will you get?
+------------------
+|pytorch-lightning-tour|
 
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :alt: PyTorchLightning neptune.ai integration
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+* see experiment as it is running,
+* log training, validation and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts and images,
+* save model checkpoints.
 
-.. code-block:: bash
+Where to start?
+---------------
+To get started with this integration, follow the :ref:`Quickstart <quickstart>` below.
+
+.. tip::
+
+    Are you a Pro User? Take a look at the :ref:`advanced options <advanced-options>`.
```

Comment:
Please change the Pro User part to something else as it sounds as if it could be a Neptune paid plan

Reply:
fixed.